bin/oneoff/data_fix/birthday_and_created_at

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env ruby
+
+# This script attempts to enforce the invariant that MONTH(birthday) == MONTH(created_at) and
+# DAY(birthday) == DAY(created_at), adjusting YEAR(birthday) as necessary, to conform to our privacy
+# policy of storing ages (not birthdays).
+
+require_relative '../../../dashboard/config/environment'
+
+updated_count = 0
+User.with_deleted.find_each do |user|
+  # Perform some console logging to give some user feedback on progress.
+  if updated_count % 1_000 == 0
+    puts "UPDATED_COUNT: #{updated_count}"
```

Comment:
Note that this may output multiple times, subsequent users are `next`ed. This seems fine to me.